src/mount/mastercomm.h

```diff
@@ -36,6 +36,8 @@
 #include "protocol/directory_entry.h"
 #include "protocol/named_inode_entry.h"
 
+inline bool masterServerIsDisconnected;
```

Comment:
Is this global variable accessed by only one thread? If it is accesed by multiple threads, then we need to protect it, probbaly using an atomic or mutex. Please check.

Reply:
Yes, it is accessed by only one thread. Anyway I changed the variable to an atomic bool.